From 4c19cf220cd2bca2a16773a007329594e373ccb8 Mon Sep 17 00:00:00 2001
From: Andreas Scherer <andreas_github@freenet.de>
Date: Sat, 15 Apr 2023 18:40:32 +0200
Subject: [PATCH 101/206] Adjust Makefile to new header files.

---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index f52ca3f..2df3698 100644
--- a/Makefile
+++ b/Makefile
@@ -74,13 +74,13 @@ clean:
 	rm -f *~ *.o *.c *.h *.tex *.log *.dvi *.toc *.idx *.scn *.ps core
 
 .SECONDEXPANSION:
-mmix-pipe.o mmix-sim.o: $$(subst .o,.c,$$@)
+mmix-pipe.o mmix-sim.o: $$(subst .o,.c,$$@) mmix-io.c
 	perl -pe "s/(#define ABSTIME)/\1 `date +%s`/" -i $<
 	$(CC) $(CFLAGS) -c $<
 
-mmix-config.o: mmix-pipe.o
+mmix-config.c mmix-io.c: mmix-pipe.c mmix-mem.c
 
-mmmix:  mmix-arith.o mmix-pipe.o mmix-config.o mmix-mem.o mmix-io.o mmmix.c
+mmmix:  mmix-arith.o mmix-config.o mmix-mem.o mmix-io.o mmix-pipe.o mmmix.c
 	$(CC) $(CFLAGS) $^ -o $@
 
 mmixal: mmix-arith.o mmixal.c
-- 
2.43.0

